pkg/operator/operator.go

```diff
@@ -156,3 +167,28 @@ func getCABundle(restConfig *rest.Config) (*string, error) {
 	}
 	return ptr.String(base64.StdEncoding.EncodeToString(transportConfig.TLS.CAData)), nil
 }
+
+func getVNETGUID(cfg *auth.Config, subnetID string) (string, error) {
```

Comment:
`getVNetGUID` would be prefferred. Also, we should find a better place than operator for this (but could wait)